src/Uploadcare/File.php

```diff
@@ -84,13 +84,32 @@ public function __get($name)
     {
         if ($name == 'data') {
             if (!$this->cached_data) {
-                $this->updateInfo();
+                try {
+                    $this->updateInfo();
+                } catch (\Exception $ex) {
```

Comment:
why do we need to catch here?
if there's an error in getting file info, this should raise.